package.json

```diff
@@ -7,7 +7,8 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "prettier": "^3.2.5"
+    "prettier": "^3.2.5",
+    "syncpack": "^12.4.0"
```

Comment:
maybe you could add a script (or multiple scripts) to run the syncpack command(s) that you recommend to use?

I am not familiar with syncpack, is it `list` and maybe `fix-mismatches`?

Also when I run the two commands I mentioned above I get some unwanted results:
![Screenshot 2024-08-16 at 13 25 56](https://github.com/user-attachments/assets/30e5700b-75b5-40b7-91d0-f7f5836d3a72)

I think you need to add a [`.syncpackrc` file](https://jamiemason.github.io/syncpack/config/syncpackrc/ config) file to let syncpack know that it should ignore those?

Reply:
I've added the `.syncpackrc` and included a note in the readme on running Syncpack commands.